ui/admin-portal/src/app/components/util/candidate-name-num-search/candidate-name-num-search.component.html

```diff
@@ -23,6 +23,10 @@
        [editable]="false"
        (selectItem)="selectSearchResult($event, input)"
        [placeholder]="placeholderText"/>
+<!--We need this ng-template to do the highlighting (using ngb-highlight).
+But not sure why we need the t.split. Just t does OK. - JC
+See https://ng-bootstrap.github.io/#/components/typeahead/examples#template
+-->
```

Comment:
The split on ' ' means if a user searches for e.g. "Sara Mou", and the result is "Sara El Moussa", both "Sara" and "Mou" will be highlighted, even though they are non-continuous in the result string.

Reply:
Thanks!